### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by sadanand1120**
Add a step to pull all submodules

### CODE REVIEW

File: src/navigation/navigation.cc [hunk 1]
```diff
@@ -533,6 +533,7 @@ bool Navigation::GetCarrot(Vector2f& carrot) {
   //     V2COMP(robot_loc_), V2COMP(v0), V2COMP(v1), (v0 - v1).norm());
   const int num_intersections = geometry::CircleLineIntersection<float>(
       robot_loc_, params_.carrot_dist, v0, v1, &r0, &r1);
+  //TODO Bugfix: say set a nav goal -> goal reached -> switched to manual and start moving -> as soon as its gets away from the prev nav target, the following error starts continuously printing. So once nav target reached this following line should not be done.
```
Comment (joydeep-b) [inline src/navigation/navigation.cc]: You didn't want to commit this with this PR, did you?

**Review by joydeep-b — CHANGES_REQUESTED** [review]

File: src/navigation/navigation.cc (hunk 1 above)
Comment (sadanand1120) [reply src/navigation/navigation.cc]: Oh I didn't want to create a separate PR for such a little thing. Hence included it with this. Is there any issue with it?

Comment (joydeep-b) [reply src/navigation/navigation.cc]: I actually did not understand this issue, could you move this to the issue tracker? https://github.com/ut-amrl/graph_navigation/issues

**Review by joydeep-b — CHANGES_REQUESTED** [review]

File: src/navigation/navigation.cc (hunk 1 above)
Comment (sadanand1120) [reply src/navigation/navigation.cc]: Removed the todo from navigation.cc and moved it to https://github.com/ut-amrl/graph_navigation/issues/46

**Review by joydeep-b — APPROVED** [review]

**Pull request merged**